VK_V1.0/VK_V1.0/PhotoScene/PhotoCollectionViewCell.swift

```diff
@@ -0,0 +1,18 @@
+// PhotoCollectionViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка c фотографией
+final class PhotoCollectionViewCell: UICollectionViewCell {
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private var photoImageView: UIImageView!
+    @IBOutlet private var likeContainerControl: LikeControl!
```

Comment:
```suggestion
    @IBOutlet private var likeContainerControl: LikeControl!
```
зачем тянуть контрол если ты его тут не используешь?